composer.json

```diff
@@ -13,7 +13,8 @@
     "require": {
         "php": ">=5.4",
         "php-http/httplug": "^1.0.0-alpha2@dev",
-        "php-http/message-factory": "^0.3"
+        "php-http/message-factory": "^0.3",
+        "php-http/client-tools": "^0.1@dev"
```

Comment:
can we avoid this dependency? i guess if we would use the guzzle promises, or put them into a separate repo. but then its a lot of repos.